src/compressed_tensors/quantization/utils/helpers.py

```diff
@@ -64,24 +64,25 @@ def calculate_qparams(
     :param quantization_args: settings to quantization
     :return: tuple of the calculated scale(s) and zero point(s)
     """
-    min_vals = torch.min(min_vals, torch.zeros_like(min_vals))
-    max_vals = torch.max(max_vals, torch.zeros_like(max_vals))
+
     device = min_vals.device
 
     bit_min, bit_max = calculate_range(quantization_args, device)
     bit_range = bit_max - bit_min
     zp_dtype = quantization_args.pytorch_dtype()
 
     if quantization_args.symmetric:
+        min_vals = torch.min(min_vals, torch.zeros_like(min_vals))
+        max_vals = torch.max(max_vals, torch.zeros_like(max_vals))
         max_val_pos = torch.max(torch.abs(min_vals), torch.abs(max_vals))
         scales = max_val_pos / (float(bit_range) / 2)
         scales = torch.clamp(scales, min=torch.finfo(torch.float32).eps)
         zero_points = torch.zeros(scales.shape, device=device, dtype=min_vals.dtype)
     else:
         scales = (max_vals - min_vals) / float(bit_range)
-        scales = torch.clamp(scales, min=torch.finfo(torch.float32).eps)
+        scales = torch.clamp(scales, min=1e-5)
```

Comment:
we pulled this to match up with AutoAWQ's `pseudo_quantize_tensor` logic exactly, but we can probably revert this line. I think it's preferable to a hard-coded `1e-5`. What do people thing?

Reply:
I'm in favor of removing this line and using our own implementation. LC cannot simultaneously match all paper's implementations, so code-readability takes priority here

Reply:
If we're going to make this change, we need to validate our W8A8 asym model support on vLLM

Reply:
I reverted this line back to the original implementation, it had little effect on results. the rounding vs truncating was the big culprit.